src/Component/ModalTask/ModalTask.jsx

```diff
@@ -49,6 +102,12 @@ const ModalTask = ({ setModalTaskFlag }) => {
                   </div>
                   <div className="my-2">
                     <input
+                      onChange={(e) => {
+                        setTaskDetails({
+                          ...taskDetails,
+                          time: e.target.value,
+                        });
+                      }}
```

Comment:
You can extract out  all the common task change handler in one function, like below-

 const taskChangeHandler = (e) => {
    setTaskDetails({ ...taskDetails, [e.target.name]: e.target.value });
  };